sql/02-bde_schema.sql

```diff
@@ -49,56 +40,39 @@ CREATE TABLE bde.crs_action (
     mode VARCHAR(4),
     flags VARCHAR(4),
     source INTEGER,
-    audit_id INTEGER NOT NULL
+    audit_id INTEGER NOT NULL,
+    UNIQUE (tin_id, id),
+    CONSTRAINT pkey_crs_action PRIMARY KEY (audit_id)
 );
 
-ALTER TABLE ONLY bde.crs_action
-    ADD UNIQUE (tin_id, id);
-
-ALTER TABLE ONLY bde.crs_action
-    ADD CONSTRAINT pkey_crs_action PRIMARY KEY (audit_id);
-
 ALTER TABLE bde.crs_action ALTER COLUMN tin_id SET STATISTICS 500;
 ALTER TABLE bde.crs_action ALTER COLUMN att_type SET STATISTICS 500;
 ALTER TABLE bde.crs_action ALTER COLUMN ste_id SET STATISTICS 500;
 ALTER TABLE bde.crs_action ALTER COLUMN audit_id SET STATISTICS 500;
 
 ALTER TABLE bde.crs_action OWNER TO bde_dba;
 
-REVOKE ALL ON TABLE bde.crs_action FROM PUBLIC;
```

Comment:
* where did you moved these `revoke` and `grant` statements ?

Reply:
I see now. Looks like previously GH didn't rendered full diff for me. This comment is solved now.